tools/configure_esp_secure_cert.py

```diff
@@ -229,24 +234,136 @@ def generate_cust_flash_partition(c, iv, hmac_key_id, key_size, device_cert, ca_
         output_file.write(output_file_data)
         output_file.close()
 
+# @info
+#       This function generates the cust_flash partition of
+#       the encrypted private key parameters when DS is disabled.
+def generate_cust_flash_partition_no_ds(device_cert, ca_cert, cs_cert, priv_key, priv_key_pass, idf_target, op_file):
+    # Following offsets have been calculated with help of esp_secure_cert_config.h
+    METADATA_OFFSET = 0
+    DEV_CERT_OFFSET = METADATA_OFFSET + 64
+    CA_CERT_OFFSET = DEV_CERT_OFFSET + 2048
+    CS_CERT_OFFSET = CA_CERT_OFFSET + 4096
```

Comment:
I think appropriate changes need to be propagated here after we have put the `CS_CERT` under a menuconfig option.